machineProblem.py

```diff
@@ -126,6 +126,12 @@ class FirstQuestion(tkinter.Frame):
     def __init__(self, parent, controller):
         tkinter.Frame.__init__(self, parent)
         
+        f = None
+        df = pd.read_csv('questionBank.csv')
```

Comment:
what's inside the quotation (''), is the PATH to the target csv file.
if PATH is the file name only, the program assumes the csv file is in the same directory with the program.
so with this code, make sure the targeted questionBank.csv is in the same folder with machineProblem.py

Reply:
the csv file is in the same folder and the error occurred

Reply:
then you might not be running the program on the right directory.

cd <the entire PATH to where machineProblem.py is>
python3 machineProblem.py